package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pino-http",
-  "version": "4.4.0",
+  "version": "4.5.0",
```

Comment:
Please revert this, we will change when we release.

Reply:
Done, pending commit